### ISSUE TITLE
Feature Request: Improvement on Tutorial setup or how to use the app.

### ISSUE THREAD

**Issue by cooldragon12** [bricesuazo/eboto#23]

**Comment by bricesuazo** [comment]
Agree. This is priority for now. Should we use feature highlight similar to this? https://wix.github.io/react-native-ui-lib/docs/components/overlays/FeatureHighlight

**Comment by cooldragon12** [comment]
Yah, that would be nice, but I think there should be a modal that welcomes the new user, then they have a choice to do a tutorial or skip it.
